src/special/bin.js

```diff
@@ -17,7 +17,7 @@ function getBinaries(dep, dir) {
   const binMetadata = getCacheOrLoad(dep, dir);
 
   if (typeof binMetadata === 'string') {
-    return [[dep, binMetadata]];
+    return [[path.basename(dep), binMetadata]];
```

Comment:
Desired transformation here:

- Unscoped package is left unchanged, e.g. `some-command`
- Scoped package discards scope, e.g. `@foo/some-command` -> `some-command`

Reply:
I see, but wouldn't that break things like `"bin": "bin/command-name.js"`?

Reply:
It would be nice to add a test but I'm not quite sure where and how best to do that.  I haven't found existing tests for this particular feature I could base a new test on.